tests/rbd/test_rbd_immutable_cache.py

```diff
@@ -0,0 +1,225 @@
+"""Test case covered -
+    CEPH-83575581 - Configuring the ceph-immutable-object-cache daemon
+
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files
+
+    Test Case Flow:
+    1. Create RBD based pool and an Image
+    2. Enable the immutable cache client settings
+    3. Install the ceph-immutable-object-cache package
+    4. Create a unique Ceph user ID, the keyring
+    5. Enable the ceph-immutable-object-cache daemon with created client
+"""
+import time
+
+from ceph.utils import get_node_by_id
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+ceph_conf_path = "/etc/ceph/ceph.conf"
+immutable_cache_path = "/mnt/immutable-cache/"
+
+
+def execute_command(node, command):
+    result = node.exec_command(cmd=command, sudo=True)
+    if result[1]:
+        log.error(f"Error executing command: {command}\n{result[1]}")
```

Comment:
An error similar to this is already logged by node.exec_command method I guess right? Why do you need a separate method here again to log this?

Reply:
used existing exec command.